server/config/config.go

```diff
@@ -779,6 +825,13 @@ func (man Manager) LoadConfig() FleetConfig {
 			},
 		},
 	}
+
+	// ensure immediately that the async config is valid for all known tasks
+	for task := range knownAsyncTasks {
+		cfg.Osquery.AsyncConfigForTask(task)
```

Comment:
Should we change `LoadConfig` to return an `error` instead of relying on `panic`?

Reply:
Ah, nvm, just saw your comment below :)